festflow/admin.py

```diff
@@ -1,8 +1,32 @@
 from django.contrib import admin
+from django.conf import settings
+from django.core.mail import EmailMessage
 from .models import *
 # Register your models here.
+
+def send_email(modeladmin, request, queryset):
+    our_email = settings.DEFAULT_FROM_EMAIL
+    users = Subscription.objects.all()
+    for newsletter in queryset:
+        for user in users:
+            email = EmailMessage(
```

Comment:
@nkprince007 no hurry. I guess send_mass_mail() will be better? What do you suggest?